backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -116,7 +117,8 @@ class CardMutationService {
         dfe: DataFetchingEnvironment,
         project: String,
         encodedCardInfos: List<String>,
-        generateStaticCodes: Boolean
+        generateStaticCodes: Boolean,
+        applicationId: Int? = null
```

Comment:
It's not clear here, why there is (only) one applicationId for this endpoint (instead of one for every card info). Please add a comment in the GraphQLDescription.

Reply:
Good point, thanks you. I renamed the param to be more clear.